### DISCUSSION TITLE
Capture (the) Point mode and how we should implement it

### DISCUSSION THREAD

**Discussion by kassibuss** [NeotokyoRebuild/neo#684] [Ideas]
Since we do not want people to get bored and we are already adding a bunch of gamemodes, then we might as well add a capture point mode.
The issue is what of kind capture point it should be. We can go with the UTNT style of C(t)P, but apparently it is flawed (@Agiel for an explanation again) and not that fun.

There is also the Team Fortress style (either 3/5CP and Attack/Defend), although if we wanted the latter, we should suit it to better fit NT, e.g teams switch between attacker/defender each round or not having to capture points in sequences(unless that is fine?)

Lastly, could just do like COD did it and have a point limit and your team gains points by capturing and holding points. Probably the easiest and simplest for (casual) play as well.

**Comment by blaberry** [comment]
There's a couple of different points being made, and I'm a little sleepy so I might comingle them

CtP sounds very similar to KOTH?

3/5CP, feels like we should just have the entities and let map makers decide how many CPs per map.

I'm assuming we're doing sideswitches every other round?

**Reply by kassibuss** [reply]
Switching sides would be a thing, I would assume yeah. I brought up the amount of points, because UTNT only had two capture points. Letting mappers decide how many caps there wouldnt be bad.
